src/main/java/com/onarandombox/MultiversePortals/listeners/MVPBlockListener.java

```diff
@@ -52,17 +48,4 @@ public void blockFromTo(BlockFromToEvent event) {
             event.setCancelled(true);
         }
     }
-
-    @EventHandler
-    public void blockPhysics(BlockPhysicsEvent event) {
-        if (event.isCancelled()) {
-            return;
-        }
-        if (event.getChangedType() == Material.NETHER_PORTAL || event.getBlock().getType() == Material.NETHER_PORTAL) {
-            PortalManager pm = this.plugin.getPortalManager();
-            if (pm.isPortal(event.getBlock().getLocation())) {
-                event.setCancelled(true);
-            }
-        }
-    }
```

Comment:
I think the idea here is to make it so when you break the block around a portal it would prevent the portal material from disappearing.

Reply:
I see. Any idea why this would cause errors when breaking the portal blocks directly?

Reply:
No idea. However, I think if we do remove this event handler, we need to cause it to delete the MV portal when the portal block gets broken.

Reply:
I think I see why this is needed. There's no way to check who broke the frame. That means that we can't check permissions. For that reason, I don't think deleting the portal is a good idea. I was thinking if we do remove this event handler, an alternative would be to add a new option that fills empty portals with particles.

I'm not sure which solution I prefer... either keep this event handler (or find another way of achieving the same result), or add particles to empty portals (with an option in the config to disable them). Let me know what you think.